corehq/apps/sms/views.py

```diff
@@ -2151,12 +2151,12 @@ def page_context(self):
             )
         else:
             wa_active_backend = turn_backend.get() if turn_backend.count() else infobip_backend.get()
-            try:
-                templates = wa_active_backend.get_all_templates()
+            templates = wa_active_backend.get_all_templates()
+            if templates:
```

Comment:
Would this make sense?
```suggestion
            if templates is not None:
```

Reply:
Yes, this makes sense to me. Although it is unlikely that we'll ever have an empty list (tuple) here. the corrected statement is definitely more accurate.